### PR TITLE
chore(other): remove apple m1 docker overrides

### LINKED ISSUE #6030: 💥 [DevOps] Remove Docker Compose File for Apple M1
## 💥 DevOps ticket

Remove Docker Compose File for Apple M1.

### Todo

- remove files `docker-compose.apple-m1.override.yml` and `deployment/docker-compose.apple-m1.override.yml`
- `deployment/DOCKER_MORE_CLOSELY.md` and `CONTRIBUTING.md`
  - move production seeding documentation to an approbriate place

### PR DESCRIPTION
**Pull request by Tirokk**
…ge in the readmes

## 🍰 Pullrequest

Remove Apple M1 Docker overrides and reduce information about its usage in the readmes.

### Issues

- fixes #6030

### Todo

- [X] None

### CODE REVIEW

File: backend/README.md [hunk 1]
```diff
@@ -79,9 +79,9 @@ More details about our GraphQL playground and how to use it with ocelot.social c
 
 ![GraphQL Playground](../.gitbook/assets/graphql-playground.png)
 
-### Database Indices and Constraints
+### Database Indexes and Constraints
```
Comment (ulfgebhardt) [inline backend/README.md:82]: indices is correct isn't it?

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -174,34 +174,15 @@ Please copy and paste the following quotes for the languages:
 
 #### Environment Variable For Apple M1 Platform
 
-To set the Docker platform environment variable in your terminal tab, run:
+To set the following environment variable seems not be needed anymore, probably because Docker knows in its newest version on which CPU it runs and sets that to default.
```
Comment (ulfgebhardt) [inline CONTRIBUTING.md:177]: I am not a fan of guesswork in the documentation

File: backend/README.md [hunk 3]
```diff
@@ -79,9 +79,9 @@ More details about our GraphQL playground and how to use it with ocelot.social c
...
 
-Database indices and constraints need to be created when the database and the
+Database indexes and constraints need to be created when the database and the
```
Comment (ulfgebhardt) [inline backend/README.md:84]: same

File: backend/README.md [hunk 4]
```diff
@@ -90,6 +90,12 @@ backend is running:
 ```bash
 # in main folder while docker-compose is running
 $ docker exec backend yarn run db:migrate init
+
+# only once: init admin user and create indexes and constraints in Neo4j database
```
Comment (ulfgebhardt) [inline backend/README.md:94]: indices

File: backend/README.md [hunk 5]
```diff
@@ -90,6 +90,12 @@ backend is running:
...
+# for development
+$ docker compose exec backend yarn prod:migrate init
+# in production mode use command
+$ docker compose exec backend /bin/sh -c "yarn prod:migrate init"
```
Comment (ulfgebhardt) [inline backend/README.md:98]: how is this section related to apple m1?

File: backend/README.md [hunk 6]
```diff
@@ -126,7 +132,7 @@ To reset the database run:
 $ docker exec backend yarn run db:reset
 # you could also wipe out your neo4j database and delete all volumes with:
 $ docker-compose down -v
-# if container is not running, run this command to set up your database indeces and contstraints
+# if container is not running, run this command to set up your database indexes and constraints
```
Comment (ulfgebhardt) [inline backend/README.md:135]: indices

File: deployment/DOCKER_MORE_CLOSELY.md [hunk 7]
```diff
@@ -2,32 +2,8 @@
...
-# for production
-$ docker compose -f docker-compose.yml -f docker-compose.apple-m1.override.yml up
-
-# for production testing Docker images from DockerHub
-$ docker compose -f docker-compose.ocelotsocial-branded.yml -f docker-compose.apple-m1.override.yml up
-
-# only once: init admin user and create indexes and contraints in Neo4j database
-$ docker compose exec backend /bin/sh -c "yarn prod:migrate init"
-```
+To get more information about the Apple M1 platform you find our documentation in our main code, [here](/CONTRIBUTING.md#docker-more-closely).
```
Comment (ulfgebhardt) [inline deployment/DOCKER_MORE_CLOSELY.md:5]: can we not remove the whole file? Whats the benefit if it just referes to other files and holds no content

File: backend/README.md (hunk 1 above)
Comment (Tirokk) [reply backend/README.md:82]: Indexes is correct says Neo4j, see headline:
https://neo4j.com/docs/cypher-manual/current/indexes-for-search-performance/

File: backend/README.md (hunk 3 above)
Comment (Tirokk) [reply backend/README.md:84]: see above

File: backend/README.md (hunk 4 above)
Comment (Tirokk) [reply backend/README.md:94]: see above

File: backend/README.md (hunk 5 above)
Comment (Tirokk) [reply backend/README.md:98]: I move the command from the deleted Apple M1 section to a fitting place to have it in the docu.

File: backend/README.md (hunk 6 above)
Comment (Tirokk) [reply backend/README.md:135]: see above

File: deployment/DOCKER_MORE_CLOSELY.md (hunk 7 above)
Comment (Tirokk) [reply deployment/DOCKER_MORE_CLOSELY.md:5]: I have left it here as a hint, because someone who works on a deployment will probably not have a look into the contributions readme to find how one can dive into the Docker layer. Or?

File: CONTRIBUTING.md [hunk 8]
```diff
@@ -174,34 +174,15 @@ Please copy and paste the following quotes for the languages:
...
+But we leave this command here to keep the knowledge about how to set the platform:
```
Comment (ulfgebhardt) [inline CONTRIBUTING.md:178]: If you encounter trouble building the docker containers on an Apple M1 chip you can try to explicitly define the target platform docker builds and pulls images for:

File: deployment/DOCKER_MORE_CLOSELY.md (hunk 7 above)
Comment (Tirokk) [reply deployment/DOCKER_MORE_CLOSELY.md:5]: I deleted it @ulfgebhardt

File: CONTRIBUTING.md (hunk 8 above)
Comment (Tirokk) [reply CONTRIBUTING.md:178]: Cool text!
Done

File: CONTRIBUTING.md (hunk 2 above)
Comment (Tirokk) [reply CONTRIBUTING.md:177]: Done

**Review by ulfgebhardt — APPROVED** [review]
